pageserver/src/tenant/timeline/eviction_task.rs

```diff
@@ -263,28 +263,19 @@ impl Timeline {
             Ok(results) => results,
         };
         assert_eq!(results.len(), candidates.len());
-        for (l, result) in candidates.iter().zip(results) {
+        drop(candidates);
```

Comment:
Confirmed, dropping it is insignificant (with latest iteration).

> Or just because it's a decent-sized allocation?

Layer is Arc<LayerInner> which is just a single pointer.

Reply:
355ea43ac